openedx/adg/lms/webinars/tests/test_helpers.py

```diff
@@ -197,3 +205,70 @@ def test_remove_emails_duplicate_in_other_list(emails, reference_emails, expecte
     Test that only the list of emails that are not present in reference list of emails are returned
     """
     assert sorted(remove_emails_duplicate_in_other_list(emails, reference_emails)) == sorted(expected_emails)
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize(
+    'template, msg_id_field_name', [
+        (MandrillClient.WEBINAR_TWO_HOURS_REMINDER, STARTING_SOON_REMINDER_ID_FIELD_NAME),
+        (MandrillClient.WEBINAR_ONE_WEEK_REMINDER, ONE_WEEK_REMINDER_ID_FIELD_NAME),
+    ]
+)
+def test_save_scheduled_reminder_ids(template, msg_id_field_name):
+    """
+    Tests save_scheduled_reminder_ids method is saving reminder ids properly.
+    """
+    registration = WebinarRegistrationFactory()
+
+    mock_response = [
+        {
+            '_id': FAKE_MANDRILL_MSG_ID,
+            'email': registration.user.email
+        }
+    ]
+    save_scheduled_reminder_ids(mock_response, template, registration.webinar.id)
+    registration.refresh_from_db()
+    assert getattr(registration, msg_id_field_name) != ''
```

Comment:
1. Lets rename `mock_response` to `expected_response` since it is not a mocked response, mocked response should involve python mock object

2. Do we need `refresh_from_db` usually, we do not

3. Lets logically divide the test into three parts using line space 

```
    registration = WebinarRegistrationFactory()
    mock_response = [
        {
            '_id': FAKE_MANDRILL_MSG_ID,
            'email': registration.user.email
        }
    ]

    save_scheduled_reminder_ids(mock_response, template, registration.webinar.id)

    registration.refresh_from_db()
    assert getattr(registration, msg_id_field_name) != ''
```

Reply:
Yes `refresh_from_db` is required. The object is updated so we are checking for the updated object from db.